content/en/docs/setup/install/virtual-machine/index.md

```diff
@@ -115,14 +79,33 @@ The Istio control plane must be installed with virtual machine integration enabl
     $ istioctl install -f "${WORK_DIR}"/vmintegration.yaml
     {{< /text >}}
 
+## Create Namespace
+
+1. Create Namespace that will host the virtual machine
+
+    {{< text bash >}}
+    $ kubectl create namespace "${VM_NAMESPACE}"
+    {{< /text >}}
+
+1. Create a serviceaccount for virtual machine
+
+    {{< text bash >}}
+    $ kubectl create serviceaccount "${SERVICE_ACCOUNT}" -n "${VM_NAMESPACE}"
+    {{< /text >}}
+
 ## Create files to transfer to the virtual machine
 
-1. Make a copy of files to copy to the virtual machine
+1. Create Kubernetes token in the example we set the token expire time to 1 day
```

Comment:
@liminw I thought the whole point of this was to use short lived tokens? 1 day is not really short.

Reply:
yes we can set it to about 10 minutes that is ok